utils/network.go

```diff
@@ -174,20 +181,53 @@ func DoNetworking(args *skel.CmdArgs, conf NetConf, result *current.Result, logg
 	return hostVethName, contVethMAC, err
 }
 
+var errFileExists = fmt.Errorf("file exists")
```

Comment:
:(

I guess this is the most robust way we can check this right now?

Reply:
yes, I checked if netlink library has any builtin error types for this, but it doesn't have any for this